src/js/pages/Profile/index.scss

```diff
@@ -8,4 +8,8 @@
     overflow: hidden;
     border-radius: 10px;
   }
+
+  .btn {
+    padding: 11px;
```

Comment:
Bootstrap had `.p-x` (where x = 1, 2, 3, etc) if you prefer. Disregard if you have a lot of buttons that would require copying the class a lot